### ISSUE TITLE
"Chunk must be locked before access" error on saving

### ISSUE THREAD

**Issue by Nuckerr** [hollow-cube/polar#17]
When I add the following line (after saving the world once), it gives me this error when it next saves
```kt
		instance.setBlock(0, 42, 0, Block.STONE)
```
```
Exception in thread "main" java.lang.AssertionError: Chunk must be locked before access
	at net.minestom.server.instance.DynamicChunk.assertLock(DynamicChunk.java:320)
	at net.minestom.server.instance.DynamicChunk.getBlock(DynamicChunk.java:149)
	at net.hollowcube.polar.PolarLoader.updateChunkData(PolarLoader.java:310)
	at net.hollowcube.polar.PolarLoader.lambda$saveChunks$3(PolarLoader.java:253)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at net.hollowcube.polar.PolarLoader.saveChunks(PolarLoader.java:253)
	at net.minestom.server.instance.InstanceContainer.saveChunksToStorage(InstanceContainer.java:241)
	at wtf.nucker.buildstom.BuildstomServer.setupWorld$lambda$2(BuildstomServer.kt:60)
	at net.minestom.server.timer.SchedulerImpl.handleTask(SchedulerImpl.java:92)
	at net.minestom.server.timer.SchedulerImpl.submitTask(SchedulerImpl.java:68)
	at net.minestom.server.timer.SchedulerManager.submitTask(SchedulerManager.java:25)
	at wtf.nucker.buildstom.BuildstomServer.setupWorld(BuildstomServer.kt:58)
	at wtf.nucker.buildstom.BuildstomServer.<init>(BuildstomServer.kt:39)
	at wtf.nucker.buildstom.MainKt.main(Main.kt:4)
	at wtf.nucker.buildstom.MainKt.main(Main.kt)
```
The error points to this code:
```kt
			buildInstance.saveChunksToStorage()
```
And removing the set block code removes all errors. This happens with a completely new world and also with the file saved from that new world

**Comment by mworzala** [comment]
Hmmm probably this is an issue but I have not run it in with assertions enabled to hit that assertion, I will take a look

**Comment by Nuckerr** [comment]
I've found the problem, this issue occurs when I have TNT also added as a dependency. Could be an issue on their end?

**Comment by Nuckerr** [comment]
Just to clarify, polar is the world loader, and there is no TNT code, it's not even imported into the class. It's just added as a dependency in the codebase

**Comment by mworzala** [comment]
I don't think this is related to TNT

**Comment by mworzala** [comment]
Fixed in [1.3.1](https://github.com/hollow-cube/polar/releases/tag/1.3.1), should be available on maven central in a little bit

**Comment by Nuckerr** [comment]
Ah, thanks, sorry for any confusion
